src/static/add_tta_match_result.html

```diff
@@ -11,7 +11,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width">
   <script src="/static/js/http.js" async=true></script>
-  <script src="/static/js/add_ffa_match_result.js" async=true></script>
+  <script src="/static/js/add_tta_match_result.js" async=true></script>
```

Comment:
create add_tta_match_result.js as an empty file in the same pull request?

Reply:
I plan to start with a functional /add_tta_match_result, and then incrementally modify the code to support TTA-specific logic. I tried and when add_tta_match_result.js is empty, /add_tta_match_result is not functional.

If you think it's better to build from scratch for this handler, then I can make add_tta_match_result.js an empty file. What do you think?

Reply:
Make sense, then maybe copy the add_ffa_match_result.js to add_tta_match_result.js?

Reply:
done